config/default.yml

```diff
@@ -2858,7 +2858,7 @@ Naming/InclusiveLanguage:
   Description: 'Recommend the use of inclusive language instead of problematic terms.'
   Enabled: false
   VersionAdded: '1.18'
-  VersionChanged: '1.49'
+  VersionChanged: <<next>>
```

Comment:
These seem more like bug fixes rather than specification changes. If so, this update should be unnecessary.

Reply:
I wasn't so sure about if this actually qualifies for a change. Moved to bugfix instead